### PR TITLE
Update schedule to show openstreetmap, hkcota and ibm talks

### PR DESCRIPTION
**Pull request by railsstudent**
#110

### CODE REVIEW

File: assets/data/sessions.yml [hunk 1]
```diff
@@ -22,12 +22,14 @@ day-1:
     timeslot: slot-3
     venue: function-room
     topic: live-dns-new-gTLD
-  - name: 45-min Keynote - TBC (IBM)
+  - name: 45-min Keynote - Rapidly build Cognitive applications with Bluemix, a cloud with Open Source at its heart (IBM)
```
Comment (tonyhhyip) [inline assets/data/sessions.yml]: Just use 45-min Keynote, topic field will do the rest

**Review by tonyhhyip — CHANGES_REQUESTED** [review]

**Comment by railsstudent** [comment]
Fixed. Please review

File: assets/data/sessions.yml [hunk 2]
```diff
@@ -175,9 +177,10 @@ day-2:
     timeslot: slot-13
     venue: training-room-1
     row: 3
-  - name: 30-min Talk - TBC
+  - name: 30-min Talk - How OpenStreetMap makes a different in Taiwan and the challenges the project faced in East Asia
```
Comment (tonyhhyip) [inline assets/data/sessions.yml]: Just 30-min Talk

File: assets/data/sessions.yml [hunk 3]
```diff
@@ -22,12 +22,14 @@ day-1:
     timeslot: slot-3
     venue: function-room
     topic: live-dns-new-gTLD
-  - name: 45-min Keynote - TBC (IBM)
+  - name: 45-min Keynote
     timeslot: slot-4
     venue: function-room
-  - name: 30-min Talk - TBC
+    topic: bluemix-cognitive-application
+  - name: 30-min Talk - Better than Free Giveaways - Open Source Business Models with Examples (HKCOTA)
```
Comment (tonyhhyip) [inline assets/data/sessions.yml:29]: Just 30-min Talk

**Review by tonyhhyip — CHANGES_REQUESTED** [review]
Forget to remind on these 2

**Comment by railsstudent** [comment]
Fixed and resolved conflicts

**Review by tonyhhyip — APPROVED** [review]
Thanks

**Pull request merged**